lib/swagger.js

```diff
@@ -129,11 +130,17 @@ function addRoute(app, uri, doc, opts) {
  * @return {Object}      Resource doc.
  */
 function generateResourceDoc(opts) {
+  var apiInfo = _cloneDeep(opts.apiInfo);
+  for (var propertyName in apiInfo) {
+    var property = apiInfo[propertyName];
+    apiInfo[propertyName] = Array.isArray(property) ? property.join('') : property;
```

Comment:
We need to do the same thing for `description`, `notes`, and `summary` on endpoints, parameters, and properties